src/cnd/cnd.ts

```diff
@@ -80,6 +85,10 @@ export type LedgerAction =
   | {
       type: "ethereum-call-contract";
       payload: EthereumCallContractPayload;
+    }
+  | {
+      type: string;
```

Comment:
Why?

Reply:
So we can properly have a proper return type for `doLedgerAction` - @thomaseizinger can probably elaborate more on that :)

Reply:
Because it is actually more correct.

We lied to the typesystem before in expressing that there can never be more than these actions, which is not true. We are adding some with LN.

Expressing the type like this forces us to write code that deals with unknown actions correctly.